lib/Tools/LinearLayout.cpp

```diff
@@ -955,6 +955,17 @@ LinearLayout LinearLayout::invertAndCompose(const LinearLayout &outer) const {
   return ret;
 }
 
+LinearLayout LinearLayout::invert() const {
```

Comment:
Bikeshedding. Should be call it `inverse`? as it's returning the inverse?

Also, unrelated. Not for this PR but leaving this idea out there. I think there's a space to implement `LinearLayout::pseudoinverse` as a less restrictive version of this function (and less safe). It would have the same implementation without the assert, and `inverse` would just assert that's invertible and then return `pseudoinverse`. This `pseudoinverse` could be useful to @Mogball if he wants to support broadcasting in the srcLayout in `gather`.

Reply:
`inverse` is an adjective while `invert` is a verb. That's the only reason I chose `invert` to make it more consistent with other method names in Linear Layout.

Reply:
I vote for `inverse` as that's what it returns, but not a hill I'm willing to die on

Reply:
Jeff voted for `invert` so I'll leave it unchanged now